src/components/MapView.js

```diff
@@ -11,6 +13,18 @@ const Map2 = ReactMapboxGl({
 const speciesColorMap = Map({'black bear':'#801e78','bobcat': '#498029','coyote': '#561480','cougar/mountain lion': '#802a00','raccoon': '#093c80','opossum': '#FFDC26','river otter': '#7083ff'});
```

Comment:
You may use this as reference for the first TODO in the description. I created this map from the initial mocks.

Reply:
Yes! Amazing, thank you. Will update soon 🙂